components/Pagination.js

```diff
@@ -168,7 +169,8 @@ class Pagination extends Component {
           <button
             className="bx--pagination__button bx--pagination__button--backward"
             onClick={this.decrementPage}
-            disabled={this.props.disabled || statePage === 1}>
+            disabled={this.props.disabled || statePage === 1}
```

Comment:
Want to add a quick test for this case and the case below? Just to help protect against any regressions.

Reply:
👍